### PR TITLE
Reference: List all user last names

### PR DESCRIPTION
**Pull request by skym97**
Assignment 7.1

### CODE REVIEW

File: docs/api/list_all_user_last_names.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### Operations: 
+
+  * ' GET /users/last_names ': Retrieves the last names of all users.
+
+#### Examples: 
+
+Request:
+
+curl -X GET https://api.example.com/users/last_names
```
Comment (rbwatson) [inline docs/api/list_all_user_last_names.md:27]: I tried this and got:
```
$ curl -X GET https://api.example.com/users/last_names
curl: (6) Could not resolve host: api.example.com
```

I then tried the local URL and got:
```
$ curl -X GET http://localhost:3000/users/last_names
Not Found
```
Are you sure this is the correct syntax and that this operation is supported?

File: docs/api/users-get-all-users.md [hunk 2]
```diff
@@ -50,3 +50,7 @@ None
 | ------------- | ----------- | ----------- |
 | 200 | Success | Requested data returned successfully |
 |  ECONNREFUSED | N/A | Service is offline. Start the service and try again. |
+
+### Additional Links
+
+[List all user last names](api/list_all_user_last_names.md)
```
Comment (rbwatson) [inline docs/api/users-get-all-users.md]:
```suggestion
[List all user last names](list_all_user_last_names.md)
```
The URL in your file is a relative URL, so the link that results from your code is:

```
https://rbwatson.github.io/to-do-service-public/api/api/list_all_user_last_names.md
```

If the file that you're linking to is in the same folder as you're linking from, all you need to put is the filename.

**Review by rbwatson — CHANGES_REQUESTED** [review]
You might need to modify your topic some. I don't think that this feature/syntax is supported.

Perhaps try something like, "Get users by last name" using `GET {base_url}/users?last_name=<name-to-find>`. If `last_name` is already documented, you could use `first_name`.

After you update your topic, be sure to squash your commits and build the docs from your feature branch to test everything.

**Review by rbwatson — CHANGES_REQUESTED** [review]
Rebased the branch to review.
Please update the topic to document a supported feature. For example, Get users by last_name.
And then test, squash, and request a review of the new topic.

**Comment by rbwatson** [comment]
I didn't see that you were working on this.
I just rebased your pull-request/feature branch, so be sure to update your fork & clone, before you continue.
Sorry for the confusion!
--bob

**Comment by skym97** [comment]
Apologies for the delay. Was sick the past couple of weeks with other things I needed to tend to (while being sick on top of that).

I made the changes requested (I believe) and I hope I rebased correctly with the edits. Please let me know if I haven't.

**Review by rbwatson — APPROVED** [review]
Rebased and squashed for merge.
Updated links to pass link-check.

**Pull request merged**